docs/query-language.md

```diff
@@ -341,6 +341,14 @@ have the attribute specified by `field` with one of the values specified in
 
 **Result Type:** array of key/count objects
 
+**Caveats:**
+* TopN on a frame with ranked cache will return the top bitmaps sorted by count.
+* Non-ranked frames maintain a LRU (least recently used) cache, thus a TopN query on this frame will return bitmaps sorted in most recently setbit order.
+* The frame's cache size determines the limit of sorted bitmaps.  
+* The ranked cache maintains a sorted set of bitmaps by count up to the cache size.  
+* Once full, the cache will truncate the set of bitmaps at the frame options CacheSize limit.  Bitmaps with a similar count that straddle the limit will be truncated in no particular order.
```

Comment:
Once full, the cache will truncate the set of bitmaps according to the frame option `CacheSize`.  Bitmaps that straddle the limit and have the same count will be truncated in no particular order.